packages/connect-ui/src/views/Passphrase.tsx

```diff
@@ -31,15 +32,32 @@ const Divider = styled.div`
     background: ${props => props.theme.STROKE_GREY};
 `;
 
-export type PassphraseProps = Extract<UiEvent, { type: 'ui-request_passphrase' }> & {
-    onPassphraseSubmit: (value: string, enterOnDevice?: boolean) => void;
-};
+export type PassphraseEventProps = Extract<UiEvent, { type: 'ui-request_passphrase' }>;
+
+type PassphraseProps = PassphraseEventProps & { postMessage: any };
```

Comment:
`any` is something like 

```   postMessage: (message: CoreMessage) => void;
```
also this type is already defined in few places. couldn't we share that?

Reply:
7810de4f33b02710c116180a66e801a69a0c699a